docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+styleguide:
+  build: .
+  volumes:
+    - ./bin:/app/bin
+    - ./src:/app/src
+    - ./projects:/app/projects
+  env_file:
+    - .env
+  ports:
```

Comment:
If we use on an `expose` directive here, then I believe you can reference the docker host via the dinghy proxy as `http://styleguide.prx.docker`  (instead of http://styleguide.prx.docker:6006  [unless I'm getting ports/expose mixed up!!  it may work either way])

Reply:
Can confirm that the dinghy proxy does work with this setup!